### ISSUE TITLE
Include Deposit Products endpoints

### ISSUE THREAD

**Issue by NAlexPear** [unit-finance/unit-node-sdk#412]
In the dashboard, deposit product information for an org is populated from the `/deposit-products` and `deposit-products/:id` endpoints. Those are useful to callers, too, especially when handling [`account.updated`](https://docs.unit.co/events/#accountupdated) events.

**Comment by Yaellever** [comment]
@NAlexPear  I am not sure I understand the request. can you please specify what exactly is missing from the SDK?

**Comment by NAlexPear** [comment]
@Yaellever no, it's a currently-undocumented route called `/deposit-products`, called internally by the dashboard to populate information about each deposit product. We can query that route just fine directly, but it would be nice to get a convenience method for that endpoint wrapped up in this client, too.
